samples/snippets/quickstart_analyzeiampolicylongrunning_test.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+from google.cloud import bigquery
+from google.cloud import storage
+
+import pytest
+
+import quickstart_analyzeiampolicylongrunning
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+BUCKET = "analysis-{}".format(uuid.uuid4().hex)
+DATASET = "analysis_{}".format(int(uuid.uuid4()))
```

Comment:
Just curious. Why do you use `.hex()` for the `BUCKET` and `int()` for the `DATASET`?

Reply:
I think bucket and dataset has a limit set of characters we can use to name them. Using those  prevent from hitting naming format issues.

Reply:
Why do you use different method for each? Can you just use one for both?

Reply:
Done.